code/2_analysis/scatterplots/figA1b.do

```diff
@@ -0,0 +1,31 @@
+* load grace data
+use "data/input/grace/grace.dta", clear
+
+* restrict to arable land
+drop if (past_area_frac == 0 & sage_crop_area_frac == 0) | ///
+	(past_area_frac == . & sage_crop_area_frac == .) | ///
+        (past_area_frac == . & sage_crop_area_frac == 0) | ///
+        (past_area_frac == 0 & sage_crop_area_frac == .)
+
+* restrict to cells with valid delta_tws
+drop if delta_wd_cm_yr==.
```

Comment:
why is this not in other scripts cleaning/using the `grace.dta` file?

Reply:
I think all analysis scripts using `grace.dta` (but not the merged datasets) should include it. Let me know if you spot any script that doesn't! I will double check too!